core/server/models/user.js

```diff
@@ -293,6 +301,8 @@ User = ghostBookshelf.Model.extend({
 
         if (status === 'active') {
             query.query('whereIn', 'status', activeStates);
+        } else if (status === 'inactive') {
```

Comment:
I think this is a bug/error, I think the code is meant to be:

```
if (status === 'active') {
     query.query('whereIn', 'status', activeStates);
} else if (status !== 'all') {
     query.query('where', {status: status});
}
```
That is the `options.status` is wrong.